Changed ALL instances of "CloudHub Fabric" to "Clustering". (#226)
mulesoft/docs-runtime-manager: modules/ROOT/pages/cloudhub-architecture.adoc

[NOTE]
Along with less available memory, workers that use a https://aws.amazon.com/ec2/instance-types/#burst[fraction of a vCore] have more limited CPU and IO, which are usually only appropriate for smaller workloads.

> With xref:cloudhub-fabric.adoc[Clustering], you can also scale your applications horizontally by adding multiple workers and load distribution over queues. For more information about scaling workers, see xref:cloudhub-fabric.adoc[Worker Scaleout]. For more information about using persistent queues on CloudHub, see xref:cloudhub-fabric.adoc[Persistent Queues].

Reviewer: Hmm...when I search for official product names, I don't see CloudHub Fabric...which at least sounds like a name. "Clustering," I fear, does not; I think we have to lowercase it, unless it is really Anypoint Platform Clustering?
(Ah, it was a service, not a product.)

Reviewer: I ran a search, and cloudhub fabric took me to clustering: common noun. GLOBAL
https://docs.mulesoft.com/runtime-manager/cloudhub-fabric

Reviewer: ```suggestion
With xref:cloudhub-fabric.adoc[clustering], you can also scale your applications horizontally by adding multiple workers and load distribution over queues. For more information about scaling workers, see xref:cloudhub-fabric.adoc[Worker Scaleout]. For more information about using persistent queues on CloudHub, see xref:cloudhub-fabric.adoc[Persistent Queues].
```

[TIP]

> You can xref:worker-monitoring.adoc[monitor your workers] and verify that they are all operating well. You can also enable automatic restarts in case any issue occurs with one.

Reviewer: "operating well"? How about "operating properly"? and "in case any issue occurs with one" could be more clear as well...:
```suggestion
You can xref:worker-monitoring.adoc[monitor your workers] to verify that they are all operating properly, and you can enable automatic restarts if one or more workers is not.
```

== CloudHub Availability and Scalability

> CloudHub has been designed to be highly available and scalable through redundancy, intelligent healing, and zero downtime updates. It also provides customers with the ability to scale and have added reliability through xref:cloudhub-fabric.adoc[Clustering].

Reviewer: ```suggestion
CloudHub is designed to be highly available and scalable through redundancy, intelligent healing, and zero-downtime updates. It also enables customers to scale and benefit from added reliability through xref:cloudhub-fabric.adoc[clustering].
```

CloudHub supports updating your applications at runtime so end users of your HTTP APIs experience zero downtime. While your application update is deploying, CloudHub keeps the old version of your application running. Your domain points to the old version of your application until the newly uploaded version is fully started. This allows you to keep servicing requests from your old application while the new version of your application is starting.

=== Clustering

Clustering provides scalability, workload distribution, and added reliability to applications on CloudHub. These capabilities are powered by CloudHub's scalable load-balancing service,
xref:cloudhub-fabric.adoc#worker-scaleout[worker scaleout], and xref:cloudhub-fabric.adoc#persistent-queues[persistent queueing] features.

> ==== Worker Scale-Out and Data Center Redundancy

Reviewer: Not hyphenated except as compound adjective:
```suggestion
==== Worker Scale Out and Data Center Redundancy
```
At some point, we should examine the organization of this document to try to avoid nesting information to four levels....

> With xref:cloudhub-fabric.adoc[Clustering], you can add multiple workers to your application to make it horizontally scale. This also adds additional reliability. CloudHub automatically distributes multiple workers for the same application across two or more datacenters for maximum reliability.

Reviewer: ```suggestion
With xref:cloudhub-fabric.adoc[clustering], you can add multiple workers to your application to make it horizontally scale. CloudHub automatically distributes multiple workers for the same application across two or more data centers for maximum reliability.
```
Middle sentence repeated by final sentence, so suggest omitting it as indicated above.

Persistent queues also guarantee delivery of your messages; even if one or more workers or datacenters go down, persistent queues facilitate disaster recovery and provide resilience to hardware or application failures.

> For more details about worker scale-out and persistent queues, refer to xref:cloudhub-fabric.adoc[Clustering].

Reviewer: ```suggestion
For more details about worker scale out and persistent queues, refer to xref:cloudhub-fabric.adoc[Clustering].
```
